server/conn_stmt.go

```diff
@@ -564,3 +564,11 @@ func (cc *clientConn) handleSetOption(data []byte) (err error) {
 
 	return errors.Trace(cc.flush())
 }
+
+func (cc *clientConn) preparedStmt2String(stmtID uint32) string {
```

Comment:
Should we add a comment for this function if we only use it for errorMsg
or
s/ preparedStmt2String/ preparedStmt2ErrMsg

Reply:
I think we the function name should describe what it does. This function could also be used in other features, e.g. `show processlist`, so I think it is not necessary to change the name.